qiita_pet/handlers/api_proxy/studies.py

```diff
@@ -242,8 +246,20 @@ def study_files_get_req(study_id, prep_template_id, artifact_type):
     # because selected is initialized to the empty list
     file_types.insert(0, (first[0], first[1], selected))
 
+    # Create a list of artifacts that the user has access to, in case that
+    # he wants to import the files from another artifact
+    user = User(user_id)
+    artifact_options = []
+    for study, artifacts in viewitems(user.user_artifacts(
```

Comment:
Just to confirm - you wanna add to the list the artifacts of the current study - correct?